nucypher/characters/base.py

```diff
@@ -555,3 +543,6 @@ def make_cli_controller(self, crash_on_error: bool = False):
 
         self.controller = controller
         return controller
+
+    def disenchant(self):
```

Comment:
Might be picking nits, but I'm not sure I'm sold on this name - seems overly complicated 😕 . IN this spirit of your wordplay - `disassociate` or `disconnect`?

Reply:
Haha, yeah, it's not a very good word.

I think that the real thing is to just make a single `stop` and tell people they can't restart.

Reply:
I'm really enjoying the character animation metaphor unfolding here.  Perhaps simply `stop` will do but I don't mind `disenchant`... (what happens at the end of animation?) its not as flowery as "bow out" for example :-)